app/main.py

```diff
@@ -1,2 +1,66 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.capacity = 8
+        self.hash_table = [None for _ in range(self.capacity)]
+        self.size = 0
+
+    def __getitem__(self, item):
+        index = hash(item) % self.capacity
+
+        for _ in range(2):
+            for i in range(self.capacity):
+                if index + i == self.capacity:
+                    index = 0
+                if self.hash_table[index + i] is not None:
```

Comment:
Could you just call break after the first None value? 
Now you search key  in the whole list which is not optimal